README.md

```diff
@@ -6,6 +6,12 @@ for a Quick Start guide and implementation details.
 
 # Release Information
 
+### v0.13.0
```

Comment:
Only this template has different versioning rule in the official templates. I think this template should be use same versioning rule which is synchronized to PredictionIO version, but `0.13.0` is lower than the latest version `6.0` in this template. Also this template has distinctive documentation compared to others.

@dszeto How do you think about versioning of this template? Do you know why this template is so distinctive?